jsonnet/kube-prometheus/versions.json

```diff
@@ -1,12 +1,12 @@
 {
-  "alertmanager": "0.21.0",
-  "blackboxExporter": "0.18.0",
-  "grafana": "7.5.4",
-  "kubeStateMetrics": "2.0.0",
+  "alertmanager": "0.22.1",
+  "blackboxExporter": "0.19.0",
+  "grafana": "",
+  "kubeStateMetrics": "",
```

Comment:
This seems to be off :)

Reply:
Weird... it worked on my machine :thinking:

Reply:
![image](https://user-images.githubusercontent.com/6232346/120213636-b8886800-c233-11eb-9e8a-201f8efc1208.png)